.github/workflows/flutter-test.yml

```diff
@@ -3,11 +3,13 @@ on:
   push:
     paths:
       - '.github/workflows/**'
+      - 'api/**'
```

Comment:
Same question... if our Flutter tests are unit tests and not integration tests, they wouldn't depend on API changes. I think there's a chance of false positive tests that give us a false sense of security (e.g. the API could change, we don't update our Flutter tests to the new API spec, Flutter unit tests still pass, but app runs fail).

Unless we think that we'll be writing Flutter tests that actually hit the APIs, but that would be a different conversation...

Reply:
Its not about hitting the APIs.  Its that the client should compile based on protoc-generated dart classes from the protobufs in api/